test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/FurtherReturnGuidanceControllerSpec.scala

```diff
@@ -0,0 +1,183 @@
+
+  implicit lazy val messagesApi: MessagesApi = controller.messagesApi
+
+  def isValidJourney(journeyStatus: JourneyStatus): Boolean =
+    journeyStatus match {
+      case _: StartingNewDraftReturn | _: FillingOutReturn | _: StartingToAmendReturn => true
+      case _                                                                          => false
+    }
+
+  def sessionDataWithStartingNewDraftReturn(
+    multipleDisposalsAnswers: MultipleDisposalsTriageAnswers,
+    name: Either[TrustName, IndividualName] = Right(sample[IndividualName]),
+    userType: UserType = UserType.Individual,
+    previousSentReturns: Option[List[ReturnSummary]] = None,
+    representeeAnswers: Option[RepresenteeAnswers] = None
+  ): (SessionData, StartingNewDraftReturn) = {
+    val individualUserType     = multipleDisposalsAnswers.fold(_.individualUserType, _.individualUserType)
+    val startingNewDraftReturn = sample[StartingNewDraftReturn].copy(
+      newReturnTriageAnswers = Left(multipleDisposalsAnswers),
+      subscribedDetails = sample[SubscribedDetails].copy(name = name),
+      agentReferenceNumber =
+        if (userType === UserType.Agent) Some(sample[AgentReferenceNumber])
+        else None,
+      representeeAnswers = representeeAnswers.orElse {
+        if (
+          individualUserType.contains(PersonalRepresentative) || individualUserType
+            .contains(PersonalRepresentativeInPeriodOfAdmin)
+        )
+          Some(
+            sample[CompleteRepresenteeAnswers]
+              .copy(dateOfDeath = Some(sample[DateOfDeath]))
+          )
+        else if (individualUserType.contains(Capacitor))
+          Some(sample[CompleteRepresenteeAnswers].copy(dateOfDeath = None))
+        else None
+      },
+      previousSentReturns = previousSentReturns.map(PreviousReturnData(_, None))
+    )
+    val sessionData            = SessionData.empty.copy(
+      journeyStatus = Some(startingNewDraftReturn),
+      userType = Some(userType)
+    )
+    sessionData -> startingNewDraftReturn
+  }
+
+  "FurtherReturnGuidanceController" when {
+
+    "handling requests to display the further return guidance page" must {
+
+      def performAction(back: String): Future[Result] =
+        controller.guidance(back)(FakeRequest())
+
+      behave like redirectToStartWhenInvalidJourney(
+        () => performAction(FurtherReturnGuidanceController.BackLinkLocations.furtherReturnStart),
+        isValidJourney
+      )
+
+      "show an error page" when {
+
+        "the back link parameter is not understood" in {
+          inSequence {
+            mockAuthWithNoRetrievals()
+            mockGetSession(
+              sessionDataWithStartingNewDraftReturn(
+                IncompleteMultipleDisposalsTriageAnswers.empty,
+                name = Right(sample[IndividualName])
+              )._1
+            )
+          }
+
+          checkIsTechnicalErrorPage(performAction("abc"))
+        }
+
+      }
+
+      "display the page" when {
```

Comment:
it'd be good to unit test the different h1's for the different user types as well - only individual "self" users are being tested atm

Reply:
making sure the different journey statuses are covered would also be good (`StartingNewDraftReturn`, `FillingOurReturn` and `StartingAmendReturn`). It'd be enough to make sure the tests for each user type covers all the different journey statuses that are allowed. we don't necessarily need every user type with all journey status types

Reply:
Is it okay how we did for other controllers like `ReliefDetailsControllerSpec`?